app/helpers/management_table_helper.rb

```diff
@@ -0,0 +1,5 @@
+module ManagementTableHelper
+  def updated_at_time(object)
+    object.updated_at.strftime("Updated on %m/%d/%Y %I:%M%p")
+  end
+end
```

Comment:
I agree with Chris, this distance_of_time_in_words is a built in rails view helper that will do essentially the same thing.